en/docs/learn/deploying-the-sample-app.md

```diff
@@ -594,7 +594,66 @@ on your browser.
 For example,
 `                     http://localhost.com:8080/pickup-manager/home.jsp                   .         `
 
+## Deploying the photo-editor and photo-viewer webapps
 
+!!! tip "Before you begin" 
+    Install Apache Tomcat 8.x as explained in the
+    beginning of this document.
+
+    
+### Download the samples
+
+To be able to deploy a WSO2 Identity Server sample, you need to download
+it onto your machine first.
+
+Follow the instructions below to download the samples from GitHub.
+
+1. Navigate to [WSO2 Identity Server Samples](https://github.com/wso2/samples-is/releases).
+2. [Download](https://github.com/wso2/samples-is/releases/download/v4.1.0/photo-edit.war) the `photo-edit.war` file from the
+   latest release assets.
+3. [Download](https://github.com/wso2/samples-is/releases/download/v4.1.0/photo-view.war) the `photo-view.war` file from the
+   latest release assets.
+
+### Deploy the sample web app
+
+Deploy the sample web apps on a web container.
+
+1.  Copy the .war files into the `           webapps          `
+    folder. For example,
+    `           <TOMCAT_HOME>/apache-tomcat-<version>/webapps          `
+    .
+2.  Start the Tomcat server.
+
+To check the **photo-edit** sample application, navigate to
+`          http://<TOMCAT_HOST>:<TOMCAT_PORT>/photo-edit/index.jsp         `
+on your browser.
+
+For example,
+`                     http://localhost.com:8080/photo-edit/index.jsp                 .         `
+    
+### Configuring the service provider
+
+Once the application is deployed, the relevant service providers are created on the management console using dynamic client registration. Follow the steps below to view the service providers and the configurations.
+
+1.  Return to the WSO2 IS management console.
+
+2.  Navigate to **Main**>**Identity**>**Service Providers** and click **List**. Note that the `photo-editor` and `photo-view` service providers have been created using dynamic client registration. 
+
+    ![photo-samples-service-providers](../assets/img/learn/photo-samples-service-providers.png)
+
+3.  Click **Edit** to view the photo-editor and photo-view service provider configurations. 
+
+3.  Expand **Inbound Authentication Configuration** section and then expand **OAuth/OpenID Connect Configuration**. 
```

Comment:
Under the **Inbound Authentication Configuration** section, click **OAuth/OpenID Connect Configuration**.